localcoinbot.py

```diff
@@ -314,12 +334,202 @@ def gas(update, context):
     except TypeError as e:
         LOGGER.warning(f'Rate Limited - {str(e)}')
         message = "Sorry Unable to Fetch Gas Estimates Right Now..."
-        gas_msg = context.bot.send_message(
-            chat_id=update.message.chat.id,
+        gas_message = context.bot.send_message(
+            chat_id=update.effective_message.chat.id,
             text=message,
             parse_mode=ParseMode.HTML
             )
-        cleaner(context, gas_msg)
+        cleaner(context, gas_message)
+
+
+def price(update, context):
+    # Gather and Show Crypto Stats
+    remove_command(context, update)
+    price_resp = cg.get_price(
+        ids=context.args[0],
+        vs_currencies=context.args[1],
+        include_market_cap='true',
+        include_24hr_vol='true',
+        include_24hr_change='true',
+        )
+
+    base_pair = str(context.args[1]).upper()
+    crypto = str(context.args[0]).upper()
+
+    try:
+        crypto_stats = price_resp.get(context.args[0])
+
+        # Cap Decimals Places at 2 Unless Token Price is Less than 1
+        full_price = price_resp.get(context.args[0]).get(context.args[1])
+        if int(full_price) < 1:
+            only_price = full_price
+        else:
+            only_price = f'{price_resp.get(context.args[0]).get(context.args[1]):,.2f}'
+
+        market_cap = f"{crypto_stats.get('usd_market_cap'):,.2f} {base_pair}"
+        volume = f"{crypto_stats.get('usd_24h_vol'):,.2f} {base_pair}"
+        daily_change = f"{crypto_stats.get('usd_24h_change'):,.2f}%"
+
+        price_msg = context.bot.sendMessage(
+            chat_id=update.effective_message.chat.id,
+            text=messages.msg_crypto_stats.format(
+                crypto,
+                base_pair,
+                only_price,
+                base_pair,
+                volume,
+                market_cap,
+                daily_change),
+            parse_mode='HTML',
+            disable_web_page_preview=True)
+
+    except:
```

Comment:
As a general rule I recommend avoiding using exceptions to control the flow of application logic. There's some debate over this in Python, see: https://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why

The first thing I'd check is how the exception is raised. Is there a way you can check `Price == None` earlier in the code without creating an exception? If the exception gets raised through `dictionary[variable_which_doesn't_exist]` then you can probably handle the case with an if statement.

Try-except clauses are advised in any situation where some outside entity is interacting with your code (for example hitting an API, reading a file, etc), but exception handling should be for handling unpredicted things, rather than used in the normal logic flow